database/migrations/postgres/1718277940006100_environments.up.sql

```diff
@@ -1 +0,0 @@
-../sqlite/1718277940006100_environments.up.sql
```

Comment:
why was this deleted?

Reply:
This was a symbolic link to the environments migrations file for sqlite. To solve the json size limit problem, we agreed to create new migrations file for postgres, but edit the migrations file for sqlite directly. Since this file is a symbolic link, it also changes by changing the original file.
The solution to this was to remove the symbolic link, and create a file with the same name and content.